modules/lb_membership_calculator/config/install/core.entity_form_display.block_content.lb_membership_calculator.default.yml

```diff
@@ -0,0 +1,35 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - block_content.type.lb_membership_calculator
+    - field.field.block_content.lb_membership_calculator.field_block
+  module:
+    - plugin
```

Comment:
add it to module dependencies, please https://github.com/YCloudYUSA/yusaopeny_memberships/pull/6

Reply:
added